cmd/thanos/config.go

```diff
@@ -11,6 +11,7 @@ import (
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
+// server config
```

Comment:
What do you mean server?

Reply:
I originally embedded the client config for grpc here but since it was only used in `query` I moved it back out, will remove this comment

Reply:
removed